sw/Modules/Comms/Src/can_interface.c

```diff
@@ -457,20 +461,23 @@ void send_NodeStatus(void) {
 }
 
 
-void send_RawIMU(struct uavcan_equipment_ahrs_SensorIMU raw_imu){
+void can_send_ImuData(struct uavcan_equipment_ahrs_SensorIMU raw_imu){
```

Comment:
It would be best if this returned some type of error, maybe we can define some generic `SB_OK` or `SB_ERROR` types?

Even if we don't use this return type for now, we can just explicitly `(void)can_send_ImuData(***)` when we call it to discard the return type.